src/cljc/rems/common/form.cljc

```diff
@@ -245,6 +248,15 @@
                      (assoc-in [:form/fields 0 :field/placeholder :fi] ""))]
         (is (empty? (validate-form-template form languages)))))
 
+    (testing "unused values are not validated"
+      (let [form (assoc form :form/fields
+                        [{:field/title {:en "en" :fi "fi"}
+                          :field/type :header
+                          :field/max-length "-3"
+                          :field/privacy :invalid
```

Comment:
See here how `:field/privacy` can be anything if it's not supported for the field type

Reply:
... I ended up just implementing the missing feature